docs/user-guide/features/skills-overview.md

```
@@ -0,0 +1,139 @@
+- ❌ 禁止**导入**到 `agents`（避免覆盖其他工具的内容）
+- ❌ 禁止**删除** `agents` 里的 skill
+- ❌ 禁止把 `agents` 当作**同步目标**
+- ❌ 不出现在「执行器管理」和 TodoDrawer 下拉框
+
+**使用场景**：本地有 cc-connect 等工具放在 `~/.agents/skills/` 的 skill，你想让 ntd 的某个执行器也能用 → 在「Skills 同步」选 source=agents、target=claudecode，复制一份过去。
+
+## 1. 什么是 Skill
+
+一个 Skill = 一个 `SKILL.md` 文件 + 可选子目录（脚本、模板等）。结构：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)